install-latest.sh

```diff
@@ -283,6 +283,15 @@ http_download() {
 }
 http_copy() {
   tmp=$(mktemp)
+  if [ ! -w "$tmp" ];
```

Comment:
mktemp should create file, with permissions such that it is only readable and writable by its owner. However, in customer ticket (see referenced ticket), this might be likely cause of failure. Adding diagnostics to ease the triaging issue re this moving forward.

Ref: https://docs.oracle.com/cd/E36784_01/html/E36870/mktemp-1.html